## Possible Windows Error
- When trying to run DeepStream on Windows a possible error

Collecting pygobject Using cached pygobject-3.52.3.tar.gz (1.2 MB) Installing build dependencies: started Installing build dependencies: finished with status 'done' Getting requirements to build wheel: started Getting requirements to build wheel: finished with status 'done' Installing backend dependencies: started Installing backend dependencies: finished with status 'done' Preparing metadata (pyproject.toml): started Preparing metadata (pyproject.toml): finished with status 'error' error: subprocess-exited-with-error Preparing metadata (pyproject.toml) did not run successfully. exit code: 1 [21 lines of output] + meson setup C:\Users\usaas\AppData\Local\Temp\pip-install-o03ulyhm\pygobject_07632d01e75c463ab0c672df618cd1de C:\Users\usaas\AppData\Local\Temp\pip-install-o03ulyhm\pygobject_07632d01e75c463ab0c672df618cd1de\.mesonpy-wcax2v3n -Dbuildtype=release -Db_ndebug=if-release -Db_vscrt=md -Dtests=false -Dwheel=true --wrap-mode=nofallback --native-file=C:\Users\usaas\AppData\Local\Temp\pip-install-o03ulyhm\pygobject_07632d01e75c463ab0c672df618cd1de\.mesonpy-wcax2v3n\meson-python-native-file.ini The Meson build system Version: 1.9.0 Source dir: C:\Users\usaas\AppData\Local\Temp\pip-install-o03ulyhm\pygobject_07632d01e75c463ab0c672df618cd1de Build dir: C:\Users\usaas\AppData\Local\Temp\pip-install-o03ulyhm\pygobject_07632d01e75c463ab0c672df618cd1de\.mesonpy-wcax2v3n Build type: native build WARNING: Failed to activate VS environment: Could not find C:\Program Files (x86)\Microsoft Visual Studio\Installer\vswhere.exe Project name: pygobject Project version: 3.52.3 ..\meson.build:1:0: ERROR: Unknown compiler(s): [['icl'], ['cl'], ['cc'], ['gcc'], ['clang'], ['clang-cl'], ['pgcc']] The following exception(s) were encountered: Running icl "" gave "[WinError 2] The system cannot find the file specified" Running cl /? gave "[WinError 2] The system cannot find the file specified" Running cc --version gave "[WinError 2] The system cannot find the file specified" Running gcc --version gave "[WinError 2] The system cannot find the file specified" Running clang --version gave "[WinError 2] The system cannot find the file specified" Running clang-cl /? gave "[WinError 2] The system cannot find the file specified" Running pgcc --version gave "[WinError 2] The system cannot find the file specified" A full log can be found at C:\Users\usaas\AppData\Local\Temp\pip-install-o03ulyhm\pygobject_07632d01e75c463ab0c672df618cd1de\.mesonpy-wcax2v3n\meson-logs\meson-log.txt [end of output] note: This error originates from a subprocess, and is likely not a problem with pip. [notice] A new release of pip is available: 25.1.1 -> 25.2 [notice] To update, run: python.exe -m pip install --upgrade pip error: metadata-generation-failed Encountered error while generating package metadata. See above for output. note: This is an issue with the package mentioned above, not pip. hint: See above for details. Process finished with exit code 1
